From 8bbd91e9648206ecd19ec598b129433d101863c1 Mon Sep 17 00:00:00 2001
From: Lin Huang <hl@rock-chips.com>
Date: Tue, 26 Apr 2016 17:48:42 +0800
Subject: [PATCH] ARM64: dts: rockchip: kevin: enable HS400 mode on kevin board

enable HS400 mode on kevin, if found it is not stable, just
remove "mmc-hs400-1_8v" property, it will use HS200 mode instead.

Change-Id: I7c5d162de1f15bcc069134ffa228d833be2b8a02
Signed-off-by: Lin Huang <hl@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi | 19 +++----------------
 1 file changed, 3 insertions(+), 16 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi
index 2f99031aa237..ba69a950c219 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi
@@ -670,24 +670,11 @@ ap_i2c_audio: &i2c8 {
 };
 
 &sdhci {
-	status = "okay";
-
-	// HACK: supports-emmc is not mainline.
-	supports-emmc;
-
-	// HACK: limit to 50MHz to avoid tuning during bringup.
-	max-frequency = <50000000>;
-
-	// TODO: add caps?
-	//cap-mmc-highspeed;
-	//mmc-hs200-1_8v;
-	//mmc-hs400-1_8v;
-
 	bus-width = <8>;
+	mmc-hs400-1_8v;
+	supports-emmc;
 	non-removable;
-
-	vmmc-supply = <&pp3000>;
-	vqmmc-supply = <&pp1800_emmc>;
+	status = "okay";
 };
 
 &sdmmc {
-- 
2.35.3

